addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -170,9 +170,17 @@ def _get_operation_line(self, bom, qty, level):
         operations = []
         total = 0.0
         qty = bom.product_uom_id._compute_quantity(qty, bom.product_tmpl_id.uom_id)
-        for operation in bom.operation_ids:
+        for index, operation in enumerate(bom.operation_ids):
+            time_start = operation.workcenter_id.time_start
```

Comment:
Same here: if the user creates two different operations in the same work center, there must be one start/stop time for each operation. So you can remove all the new lines